scipy-1.0/scipy-optimize.tex

```diff
@@ -124,9 +126,21 @@ \subsubsection*{Nonlinear Optimization}
   \label{tab:minimize}
 \end{table}
 
-Some problems may require the use of global optimizers to find the minimum of the objective function over the set of all parameter values, especially if the objective function possesses many local minima that other solvers could get stuck in.
-\texttt{Brute} performs a grid search over the entire parameter range, with the best grid point undergoing further polishing by another solver. However, this approach becomes increasingly inefficient as the problem dimensionality increases, or if the objective function has many extrema.
-\texttt{Basinhopping} \cite{Wales1997} uses random perturbation of the parameter vector, with local minimization. The acceptance of the new coordinates is dictated by the Metropolis test typically used in Monte Carlo simulations.
-\texttt{differential\textunderscore evolution} \cite{Wormington1999,Storn1997} is a stochastic solver that improves by gradual evolution of a population of candidate solutions. In each iteration trial candidates are generated by combination of candidates from the existing population; if the trial candidates represent an improvement then the population is updated. \texttt{differential\textunderscore evolution} does not require that the objective function be differentiable.
+\paragraph{Global Minimization}
+As \texttt{minimize} only seeks a local minimum, some problems require the use of a global optimization routine to find the minimum of the objective function over the set of all decision variable values, especially if the objective function possesses many local minima that other solvers could get stuck in. % e.g. if there is concern that `minimize` could get stuck in an unacceptable local minimum?
+The \texttt{brute} function performs a grid search over the entire domain, and the best grid point is ``polished'' by a local minimizer. However, this approach becomes prohibitively expensive as the problem dimensionality increases.
+% As far as I understand how brute works, the run time does not depend on the number of extrema, as it only polishes the best one
```

Comment:
Sorry, what I wrote is probably not clear enough. Yes, `brute` only polishes the best one. What I meant was that the grid search increases as the power of N. Say there's 5 grid points per parameter.
1 params - 5 evals
2 params - 25 evals
3 params - 125 evals, etc.

Reply:
OK. In the comment, I was just explaining why I removed the italicized portion:

> However, this approach becomes increasingly inefficient as the problem dimensionality increases, _or if the objective function has many extrema_."

because that part of the statement is not accurate, as the efficiency does not depend on the number of extrema.